src/main/java/org/reaktivity/nukleus/kafka/internal/cache/KafkaCacheCursorFactory.java

```diff
@@ -505,6 +515,60 @@ public boolean test(
             }
         }
 
+        private static final class Live extends None
+        {
+            private long latest;
```

Comment:
Let's rename this from `latest` to `historical` to avoid confusion about what `latest` means as more live messages arrive from Kafka.